corehq/apps/accounting/migrations/0044_paused_and_deleted_plans.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
```

Comment:
Odd that this still get generated when it's not needed in python 3. Same for the __future__ import below.

Reply:
that should be expected until we move to django 2.x. django 1.11 still supports python 2.7 so even if the developer is using py3 at the time it still needs to assume compatibility